charts/victoria-metrics-k8s-stack/hack/sync_rules.py

```diff
@@ -98,7 +99,7 @@ def new_representer(dumper, data):
     'node-network': '.Values.defaultRules.rules.network',
     'node.rules': '.Values.defaultRules.rules.node',
     'vmagent': '.Values.vmagent.enabled .Values.defaultRules.rules.vmagent',
-    'alertmaanger': '.Values.defaultRules.rules.alertmanager',
```

Comment:
Must match the [alertgroup name](https://github.com/VictoriaMetrics/helm-charts/pull/537/files#diff-be7fe52ad55b3eee294f8f15d6bd822251f33b40f76d35f45a4a837447f4c562L24) to properly set the condition.